apps/dashboard/src/components/primitives/breadcrumb.tsx

```diff
@@ -0,0 +1,96 @@
+import { cn } from '@/utils/ui';
```

Comment:
Nit picky comment: Is breadcrumb a primitive? It's a specific component displayed in one place in the app. Should we remove it from this folder?

Reply:
I guess we will use it in other areas too, like Activity feed, Subscribers list page
`< Development / Subscribers / Biswajeet Das`